WMF Framework/ReadingList+CoreDataClass.swift

```diff
@@ -14,17 +17,21 @@ public class ReadingList: NSManagedObject {
         return existingKeys
     }
     
-    public func updateCountOfEntries() {
```

Comment:
Looks like `updateCountOfEntries` wasn't used, but do you recall if its `entry.isDeletedLocally` filtering logic isn't needed? Just curious.

Reply:
inside `updateArticlesAndEntries`  we're updating `countOfEntries` with the count of `validEntries`, to get validEntries we're filtering on `isDeletedLocally`, `let validEntries = (entries ?? []).filter { !$0.isDeletedLocally }`, line `37` in the same file

Reply:
Cool thanks!!!